webpack.config.js

```diff
@@ -75,41 +82,39 @@ module.exports = async (env, options) => {
           },
         ],
       }),
-      new ExtractTextPlugin("[name].[hash].css"),
+      new MiniCssExtractPlugin({
+        filename: '[name].[hash].css',
+      }),
       new HtmlWebpackPlugin({
-        filename: "taskpane.html",
-        template: "./src/taskpane/taskpane.html",
-        chunks: ["taskpane", "vendor", "polyfills"],
+        filename: 'taskpane.html',
+        template: './src/taskpane/taskpane.html',
+        chunks: ['taskpane', 'vendor', 'polyfills'],
       }),
       // Needed as Azure deployment expects an index.html file
       new HtmlWebpackPlugin({
-        filename: "index.html",
-        template: "./src/index.html",
-        chunks: ["index"],
+        filename: 'index.html',
+        template: './src/index.html',
+        chunks: ['index'],
       }),
       new HtmlWebpackPlugin({
-        filename: "commands.html",
-        template: "./src/commands/commands.html",
-        chunks: ["commands"],
-      }),
-      new webpack.ProvidePlugin({
-        Promise: ["es6-promise", "Promise"],
+        filename: 'commands.html',
+        template: './src/commands/commands.html',
+        chunks: ['commands'],
       }),
     ],
   };
 
   if (env.WEBPACK_SERVE) {
     config.devServer = {
-      hot: true,
-      headers: {
-        "Access-Control-Allow-Origin": "*",
+      server: {
+        type: 'https',
+        options: {
+          https:
```

Comment:
Is this working for you? For me the word desktop application is complaining:
![image](https://user-images.githubusercontent.com/5037600/232285818-644e36fa-6f3d-409a-8c09-33ec4769e1ea.png)
And the browser version is also not working because it blocks the request for security reasons. It works at least in the browser if I use https://github.com/OfficeDev/Office-Addin-TaskPane/blob/d96604622a88349277ca651dfabec8d6a2784a82/webpack.config.js#L92-L100 instead.

Reply:
Not sure why this is happening. It could be caused by an expired certificate. You can clear your old certificate by running `yarn clear:certs`

https://github.com/OfficeDev/Office-Addin-Scripts/issues/190#issuecomment-649737524

Reply:
Thanks, I guess something is wrong with my local setup. I've found a workaround by manually installing the cert and reported the issue upstream: https://github.com/OfficeDev/Office-Addin-Scripts/issues/769. @koppor is it working for you?

Anyway, it's not an issue with this PR, which is working.